web/client/src/models/file.ts

```diff
@@ -44,6 +44,10 @@ export class ModelFile extends ModelArtifact<InitialFile> {
       initial?.content ?? this.initial.content ?? ''
   }
 
+  get shortName(): string {
```

Comment:
Is this the filename without the extension? I've heard it called `basename` or more commonly `stem`.